app/experimenter/static/js/components/TimelinePopForm.js

```diff
@@ -0,0 +1,372 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Button, Container, Row, Col } from "react-bootstrap";
+import { boundClass } from "autobind-decorator";
+import { fromJS, Map } from "immutable";
+
+import DesignInput from "experimenter/components/DesignInput";
+import LabeledMultiSelect from "experimenter/components/LabeledMultiSelect";
+
+import { makeApiRequest } from "experimenter/utils/api";
+import {
+  VERSION_CHOICES,
+  PROPOSED_START_DATE_HELP,
+  PROPOSED_DURATION_HELP,
+  PROPOSED_ENROLLMENT_HELP,
+  CHANNEL_HELP,
+  POPULATION_PERCENT_HELP,
+  VERSION_HELP,
+  PLATFORM_HELP,
+  CLIENT_MATCHING_HELP,
+  COUNTRIES_LOCALES_HELP,
+} from "experimenter/components/constants";
+
+@boundClass
+class TimelinePopForm extends React.PureComponent {
+  static propTypes = {
+    slug: PropTypes.string,
+    shouldHavePopPercent: PropTypes.string,
+    allCountries: PropTypes.array,
+    allLocales: PropTypes.array,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: new Map(),
+      errors: new Map(),
+      loaded: false,
+      saving: false,
+    };
+  }
+
+  getEndpointUrl() {
+    return `experiments/${this.props.slug}/timeline-population/`;
+  }
+
+  async componentDidMount() {
+    try {
+      const data = await makeApiRequest(this.getEndpointUrl());
+      this.setState({
+        loaded: true,
+        data: fromJS(data),
+      });
+    } catch (error) {
+      console.error(error.message);
+    }
+  }
+
+  handleDataChange(key, value) {
+    if (value == "") {
+      value = null;
+    }
+    this.setState(({ data }) => ({
+      data: data.set(key, value),
+    }));
+  }
+
+  handleMultiSelectDataChange(selectedOptions, name) {
+    if (selectedOptions == null) {
```

Comment:
Should be a `===` check here?  I'm surprised the linter didn't complain about this, usually in JS you're supposed to always use `===` I think?

Reply:
I looked into it more, and if you use `==` with null, it will also check if it's undefined. Which in this case I probably want. But the others should be changed to `===`.

Reply:
Interesting, good to know!  I'm always learning about the weird dark magic of equality comparisons in JS